packages/web-api-scan-runner/src/runner/runner.spec.ts

```diff
@@ -52,6 +61,9 @@ describe(Runner, () => {
         id: 'id',
         partitionKey: 'item-partitionKey',
         batchRequestId: 'batch-id',
+        notification: {
+            scanNotifyUrl: 'scan-notify-url',
```

Comment:
you should have a test for 
1.) with notification object but no notify url
2.) with notification object & notify url
3.) without notification object.